make-pangenome-sketches.py

```diff
@@ -21,52 +23,88 @@ def main():
     )
     p.add_argument('--scaled', default=1000, type=int)
     p.add_argument('-k', '--ksize', default=31, type=int)
-    p.add_argument('-o', '--output', required=True)
+    p.add_argument('-o', '--output', required=True,
```

Comment:
Instead of creating two files from this one name, I suggest adding a separate argument ('--csv-info' or something) to create the second file explicitly, and to NOT create it if that argument isn't provided.

I have been roundly criticized, and rightly so, for doing this kind of thing in the past 😆 . `sourmash compare` and `sourmash plot` both create multiple files based off of an implicitly provided filename, and it's annoying for people to have to track down.